EFCache.Redis/RedisCache.cs

```diff
@@ -109,137 +246,211 @@ public bool GetItem(string key, out object value)
 					OnCachingFailed(e);
 				}
 			}
-
 			if (value == null) return false;
 
+			// If we are not using expiry (as indicated by values < MaxValue), return
```

Comment:
@jkdufair shouldn't this comment say `indicated by values == MaxValue`? it seems that is the logic below